packages/cli/src/controllers/workflowStatistics.controller.ts

```diff
@@ -29,7 +29,7 @@ export class WorkflowStatisticsController {
 	 */
 	// TODO: move this into a new decorator `@ValidateWorkflowPermission`
 	@Middleware()
-	async hasWorkflowAccess(req: ExecutionRequest.Get, res: Response, next: NextFunction) {
+	async hasWorkflowAccess(req: ExecutionRequest.GetOne, res: Response, next: NextFunction) {
```

Comment:
I think this is a piece of tech debt, reusing the `ExecutinRequest` type in here seems like a mistake to me wdyt?